src/scss/variables.scss

```diff
@@ -50,6 +50,7 @@
   --primary3: hsl(207, 100%, 63%);
   --primary3-hover: hsl(207, 100%, 67%);
   --primary3-active: hsl(207, 100%, 59%);
+  --primary4: hsl(214, 84%, 56%);
```

Comment:
Skąd wziął się kolor --primary4? Nie mamy takiego w palecie. Jeśli to błąd UXa, to zamiast --primary4 użyj któregoś z 3 pozostałych primary